mobile/src/main/java/rs/readahead/washington/mobile/domain/entity/Metadata.java

```diff
@@ -208,4 +214,9 @@ public String getFileHashSHA256() {
     public void setFileHashSHA256(String fileHashSHA256) {
         this.fileHashSHA256 = fileHashSHA256;
     }
+
+    public String getMetadataText() {
```

Comment:
Please remove JSON serialization code from Metadata value object. Any code needed this object serialized will deal with serialization.